src/services/infra/DynamoDBService.ts

```diff
@@ -1,82 +1,45 @@
-import {
-  DeleteCommandOutput,
-  GetCommandOutput,
-  UpdateCommandOutput,
-} from "@aws-sdk/lib-dynamodb"
+import { DeleteCommandOutput } from "@aws-sdk/lib-dynamodb"
 import autoBind from "auto-bind"
 
 import { config } from "@config/config"
 
 import { SiteLaunchMessage } from "@root/../microservices/site-launch/shared/types"
 
-import DynamoDBClient, { UpdateParams } from "./DynamoDBClient"
+import DynamoDBClient from "./DynamoDBClient"
 
-const MOCK_LAUNCH: SiteLaunchMessage = {
-  repoName: "my-repo",
-  appId: "my-app",
-  primaryDomainSource: "example.com",
-  primaryDomainTarget: "myapp.example.com",
-  domainValidationSource: "example.com",
-  domainValidationTarget: "myapp.example.com",
-  requestorEmail: "john@example.com",
-  agencyEmail: "agency@example.com",
-  githubRedirectionUrl: "https://github.com/my-repo",
-  redirectionDomain: [
-    {
-      source: "example.com",
-      target: "myapp.example.com",
-      type: "A",
-    },
-  ],
-  status: { state: "pending", message: "PENDING_DURING_SITE_LAUNCH" },
-}
 export default class DynamoDBService {
   private readonly dynamoDBClient: DynamoDBClient
 
   private readonly TABLE_NAME: string
 
-  constructor() {
-    this.dynamoDBClient = new DynamoDBClient()
+  constructor(dynamoDBClient: DynamoDBClient) {
+    this.dynamoDBClient = dynamoDBClient
     this.TABLE_NAME = config.get("aws.dynamodb.siteLaunchTableName")
     autoBind(this)
   }
 
   async createItem(message: SiteLaunchMessage): Promise<void> {
-    await this.dynamoDBClient.createItem(this.TABLE_NAME, MOCK_LAUNCH)
+    await this.dynamoDBClient.createItem(this.TABLE_NAME, message)
   }
 
-  async getItem(message: SiteLaunchMessage): Promise<GetCommandOutput> {
-    return this.dynamoDBClient.getItem(this.TABLE_NAME, MOCK_LAUNCH.appId)
-  }
+  async getAllSuccessOrFailureLaunches(): Promise<SiteLaunchMessage[]> {
```

Comment:
wdyt of changing the name to `getAllCompletedLaunches`?

Reply:
faed0d0c6cd27a6c2a5c9aa02c5c28acbf797d6c